.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -61,7 +61,7 @@ Branch comparison: https://github.com/ignitionrobotics/ign-cmake/compare/<TO_BRA
 
 **Note to maintainers**: Remember to **Merge** with commit (not squash-merge or rebase)
 
----
+🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸
 
 <!-- For maintainers only -->
 
```

Comment:
For the release, how about :arrow_up: ?

Reply:
:heavy_check_mark: but with :balloon: 4865cb95b60742e8542ab1f352f1a187ddb8d7c5